app/models/unavailable_as_candidate_for.rb

```diff
@@ -40,7 +42,22 @@ def self.describe_expiration_length
 
   def self.cleanup_expired!
     where.not(expires_at: nil).
-    where(arel_table[:expires_at].lt(Date.current)).destroy_all
+      where(arel_table[:expires_at].lt(Date.current)).
+      destroy_all
   end
 
+  def self.download_columns
```

Comment:
This is #6133 right? Some differences here from the spec (missing Warehouse ID) but maybe that's okay?

Reply:
Warehouse ID is actually `Remote ID` but it needs to be paired with `Remote Source` because the clients come from multiple sources.  So that should suffice.